CHANGELOG.md

```diff
@@ -12,10 +13,108 @@
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/r1.2/code/API_definitions/qos-profiles.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/QualityOnDemand/blob/r1.2/code/API_definitions/qos-profiles.yaml)
+
+### Added
+* Added the option to query QoS profiles available on a given device by @RandyLevensalor in https://github.com/camaraproject/QualityOnDemand/pull/318 and https://github.com/camaraproject/QualityOnDemand/pull/348
+* Added `x-correlator` header to requests and responses by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/283
+* Added security scheme and scopes for each endpoint / method by @eric-murray in https://github.com/camaraproject/QualityOnDemand/pull/295
+
+### Changed
+* Changed the query for `/qos-profiles` from a GET to a POST to support the query for profiles available on a given device by @RandyLevensalor in https://github.com/camaraproject/QualityOnDemand/pull/318
+* Updated the description of `maxDuration` by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/296
+* Updated Authorization and Authentication text to ICM release 0.2.0 wording by @eric-murray in https://github.com/camaraproject/QualityOnDemand/pull/327
+* Aligned qos-profiles further with Commonalties 0.4.0 by @hdamker in https://github.com/camaraproject/QualityOnDemand/pull/326
+  * Added documentation about "Handling of device information" within the info description. 
+  * Updated schemata for error responses according to Commonalities
+
+### Fixed
+* n/a
+
+### Removed
+* Removed `TermsOfService` and `Contact` from `info` objects (they may be added by API Providers documenting their APIs) by @RandyLevensalor in https://github.com/camaraproject/QualityOnDemand/pull/311
+
+## qod-provisioning v0.1.0
+
+qod-provisioning v0.1.0 is the first initial release of this new API. It provides the ability to set a QoS profile for a device within an access network which will be applied each time the device connects to the same access network.
+
+- API definition **with inline documentation**:
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/r1.2/code/API_definitions/qod-provisioning.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/r1.2/code/API_definitions/qod-provisioning.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/QualityOnDemand/blob/r1.2/code/API_definitions/qod-provisioning.yaml)
+
+### Added
+* Initial version of QoD Provisioning mode API by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/299
+
+### Changed
+* n/a
+
+### Fixed
+* n/a
+
+### Removed
+* n/a
+
+**Full Changelog**: https://github.com/camaraproject/QualityOnDemand/compare/v0.10.1...r1.2
+
 # r1.1
```

Comment:
Do we keep r1.1 in the changelog or should we remove the pre-release once we have the public release? It is really duplicating the same info and it does not add any real value, IMO.

Reply:
See discussion in https://github.com/camaraproject/DeviceLocation/pull/252#issuecomment-2312402603